lib/form526_backup_submission/configuration.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module Form526BackupSubmission
+  ##
+  # HTTP client configuration for the {DecisionReview::Service},
+  # sets the base path, the base request headers, and a service name for breakers and metrics.
+  #
+  class Configuration < Common::Client::Configuration::REST
+    self.read_timeout = Settings.caseflow.timeout || 20 # using the same timeout as lighthouse
+
+    ##
+    # @return [String] Base path for decision review URLs.
+    #
+    def base_path
+      Settings.form526_backup.url
+    end
+
+    ##
+    # @return [String] Service name to use in breakers and metrics.
+    #
+    def service_name
+      'Form526BackupSubmission'
+    end
+
+    ##
+    # @return [Hash] The basic headers required for any decision review API call.
+    #
+    def self.base_request_headers
+      super.merge('apiKey' => Settings.form526_backup.api_key)
```

Comment:
This shouldn't really make a difference because headers are supposed to be case-insensitive, but the Lighthouses' docs always use `apikey` instead of `apiKey`

Reply:
I have seen it both ways in the code, and probably just copy/pasted as-is from somewhere. But I will change it anyways.